styles.css

```diff
@@ -0,0 +1,100 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 48px;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 3px 7px rgba(0, 0, 0, 0.1);
+}
+
+#toolbar {
+  display: flex;
+  height: 48px;
+  justify-content: space-between;
+}
+
+#logo {
+  font-size: 1.5rem;
+  font-weight: 600;
+  color: #6070ff;
+  display: flex;
+  padding: 10px;
+  align-items: center;
+  text-decoration: none;
+}
+
+#menu-bars-container {
+  padding-right: 10px;
+  display: flex;
+  align-items: center;
+}
+
+.menu-bar {
+  width: 35px;
+  height: 5px;
+  background-color: #6070ff;
+  margin: 6px 0;
+}
+
+#headline-container {
+  margin-top: 48px;
+  padding: 0 10px;
+  height: 100%;
+  width: calc(100% - 20px);
+  min-height: 100vh;
+  background-image: url('images/background.png');
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.headline-header {
+  margin: 0;
+  padding: 0;
+  height: 300px;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+  display: flex;
+  align-items: flex-end;
+}
+
+.headline-text {
+  margin: 10px 0;
+  padding: 0;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+#connect {
+  margin: 10px 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-transform: uppercase;
+}
+
+#social-icons {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: flex-start;
+}
```

Comment:
- I think it will be better if we stick to the colors on figma as much as possible. Your social icons color are quite different from what's on Figma.

Here is what i mean 👇🏽 
| Your Social Icons  |That of Figma |
| :---:   | :---: | 
| ![soicb](https://user-images.githubusercontent.com/99246359/205323278-e8b2d28b-b5b5-4236-829d-97e56d84b2d5.PNG) |![soicg](https://user-images.githubusercontent.com/99246359/205323365-fa49b9f2-ecd5-41be-aead-52d04a79fb7c.PNG)  |


Can we kindly fix this? 🍒